### PR TITLE
Another array to base64 decoder, probably fixes issue with photos

### PR DESCRIPTION
**Pull request by skalitin**

### CODE REVIEW

File: webui/src/store/modules/user.js [hunk 1]
```diff
@@ -155,8 +155,11 @@ export default {
                                 // Optimization: convert to base64 here and not in the OrgTree component
                                 // to have ready-to-use values cached in the user object
                                 if(data) {
-                                    let base64 = btoa(String.fromCharCode.apply(null, data));
-                                    elem.photo = 'data:image/jpeg;base64,' + base64;
+                                    let reader = new FileReader();
+                                    reader.readAsDataURL(data);
+                                    reader.onloadend = function() {
```
Comment (denis1stomin) [inline webui/src/store/modules/user.js]: arrow function may be?
is it Ok to provide callback after you already started the operation? (let's change onloaded and readas lines?)

**Comment by denis1stomin** [comment]
Great! Works for me now. But please take a look at my comment before merging

**Review by denis1stomin — APPROVED** [review]

File: webui/src/store/modules/user.js [hunk 2]
```diff
@@ -154,8 +154,15 @@ export default {
                             SubjectSvc.getUserPhoto(elem.id, (done) => { done(null, token);}, data => {
                                 // Optimization: convert to base64 here and not in the OrgTree component
```
Comment (denis1stomin) [inline webui/src/store/modules/user.js]: Also I'm not sure the comment is actual now

File: webui/src/store/modules/user.js (hunk 1 above)
Comment (skalitin) [reply webui/src/store/modules/user.js]: Unfortunately it does not work on Windows :( So I have to support both Blobs and typed arrays. Will fix now.

File: webui/src/store/modules/user.js (hunk 2 above)
Comment (skalitin) [reply webui/src/store/modules/user.js]: It is in theory, but agree it is not really important - let's remove

**Pull request merged**